build-farm/platform-specific-configurations/windows.sh

```diff
@@ -117,12 +117,12 @@ then
     export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched --enable-openssl-bundling"
     if [ "${JAVA_TO_BUILD}" == "${JDK8_VERSION}" ]
     then
-      export BUILD_ARGS="${BUILD_ARGS} --freetype-version 2.5.3"
+      export BUILD_ARGS="${BUILD_ARGS} --freetype-version 2.9.1"
```

Comment:
This may clash with `build.sh` where it patches `freetype.vcxproj` (for non-OpenJ9 builds) - that patch is only appropriate for 2.8.1. I'm not sure it's needed for 2.9.1 (it isn't needed for OpenJ9 builds).